R/molecule.R

```diff
@@ -24,6 +24,11 @@ Molecule <- R6::R6Class(
           private$.startValue$value <- value
         }
       }
+    },
+    #' @field  scaleDivisor Scale divisor. Its purpose is to reduce numerical noise and to enhance computation performance.
+    #' see https://docs.open-systems-pharmacology.org/working-with-mobi/mobi-documentation/model-building-components#import-molecule-and-parameter-start-values-from-excel
+    scaleDivisor = function(value){

```

Comment:
@PavelBal Link does not work here. Do you know how to make it work (i tried with \link but this is the same